### PR TITLE
Feat: Trash Feature

### PR DESCRIPTION
**Pull request by anandsr21harshit**
- Added Trash Feature
- On deleting item, it will go to trash
- From trash we can either restore the data or delete it permanently
- console.log() added in code-base for checking and understanding errors. Will be refactored in the end.

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
clean and modular code

looks good to merge

File: keep-notes/src/components/NoteCard.jsx [hunk 1]
```diff
@@ -1,21 +1,51 @@
...
+  const{state,dispatch} = useData()
+
+  async function deleteHandler() {
+    try {
+      const response = await axios.delete(`/api/notes/${notesData._id}`, {
+        headers: {
+          authorization: token,
+        },
+      });
+      console.log(response);
```
Comment (manish591) [inline keep-notes/src/components/NoteCard.jsx:17]: remove console

**Review by manish591 — APPROVED** [review]
code is looking good, remove console, use prettier for consistent formatting, rest all good

File: keep-notes/src/components/NoteCard.jsx (hunk 1 above)
Comment (anandsr21harshit) [reply keep-notes/src/components/NoteCard.jsx:17]: I added console for debugging purpose. Will remove it at the end

**Pull request merged**